gitignore

```diff
@@ -1,2 +1,22 @@
+# Folder view configuration files
 .DS_Store
-.tags
+Desktop.ini
+
+# Thumbnail cache files
+._*
+Thumbs.db
+
+# Files that might appear on external disks
+.Spotlight-V100
+.Trashes
+
+# Compiled Python files
+*.pyc
+
+# Compiled C++ files
+*.out
+
+# Application specific files
+venv
+node_modules
+.sass-cache
```

Comment:
@evgenTraytyak, we shouldn't add project or language specific files to global gitignore file since it's error-prone: someone new will not have that global gitignore and will be able to commit `node_modules` by accident.